packages/polaris-viz/src/playground/ExternalEvents.stories.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import type {Story} from '@storybook/react';
+import type {DataSeries} from '@shopify/polaris-viz-core';
+
+import {LineChart} from '../components/LineChart';
+import {PolarisVizProvider} from '../components/PolarisVizProvider';
+import {setSingleSeriesActive} from '../utilities';
+import {useWatchActiveSeries} from '../hooks';
+
+export default {
+  title: 'polaris-viz/Playground/ExternalEvents',
+  parameters: {},
+  decorators: [(Story) => <div>{Story()}</div>],
+  argTypes: {},
+};
+
+const DEFAULT_DATA: DataSeries[] = [
+  {
+    name: 'Breakfast',
+    data: [
+      {key: 'Monday', value: 3},
+      {key: 'Tuesday', value: -7},
+      {key: 'Wednesday', value: -7},
+      {key: 'Thursday', value: -8},
+      {key: 'Friday', value: 50},
+      {key: 'Saturday', value: 0},
+      {key: 'Sunday', value: 0.1},
+    ],
+  },
+  {
+    name: 'Lunch',
+    data: [
+      {key: 'Monday', value: 4},
+      {key: 'Tuesday', value: 0},
+      {key: 'Wednesday', value: -10},
+      {key: 'Thursday', value: 15},
+      {key: 'Friday', value: 8},
+      {key: 'Saturday', value: 50},
+      {key: 'Sunday', value: 0.1},
+    ],
+  },
+  {
+    name: 'Dinner',
+    data: [
+      {key: 'Monday', value: 7},
+      {key: 'Tuesday', value: 0},
+      {key: 'Wednesday', value: -15},
+      {key: 'Thursday', value: -12},
+      {key: 'Friday', value: 50},
+      {key: 'Saturday', value: 5},
+      {key: 'Sunday', value: 0.1},
+    ],
+  },
+];
+
+const CHART_ID = 'chart_01';
+
+const Template: Story = () => {
+  const [activeName, setActiveName] = React.useState('none');
+
+  useWatchActiveSeries(CHART_ID, ({detail: {index, data}}) => {
+    if (index === -1 || data == null) {
```

Comment:
When could `data` be `null` and when could `index` be `-1`?

If it's when the series is not active (e.g. moused outside the chart) I would almost expect the entire event to be `null` or the `detail` property to be `null`.